src/styles/utils/reset.scss

```diff
@@ -0,0 +1,15 @@
+body {
+  margin: 0;
+}
+
+.visually-hidden {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  margin: -1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  overflow: hidden !important;
+  clip: rect(0,0,0,0) !important;
+  white-space: nowrap !important;
+}
```

Comment:
I don't know what did you hide, but you hided it well i couldn't find :) I guess you forgot to put the class to  h1

Reply:
haha, kinda tired ;) I had it to h1, then deleted h1, lol